packages/termux-tools/pkg

```diff
@@ -80,6 +80,7 @@ select_mirror() {
 	mirrors[11]="https://termux.librehat.com/apt/termux-main"
 	mirrors[10]="https://mirror.mwt.me/termux/main"
 	mirrors[ 9]="https://termux.sahilister.in/apt/termux-main"
+	mirrors[ 8]="https://plug-mirror.rcac.purdue.edu/termux/termux-main"
```

Comment:
16 + 15 + ... + 8 != 100. You need to adjust these values such that their sum is 100.

Reply:
I'll be honest and say I don't understand how to properly adjust them then. Could you calculate what values I should set them to? I think I understand how to adjust the other file and version bump it.

Reply:
Thanks for setting up/maintaining the mirror! Please change the weights in the list to 16, 14, 13, 12, 11, 10, 9, 8, 7